RecipeMatch/View Controllers/DetailsViewController.h

```diff
@@ -13,6 +13,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface DetailsViewController : UIViewController
 @property (strong, nonatomic) SavedRecipe *savedRecipe;
```

Comment:
i'm not sure `savedRecipe` is actually needed, or used for anything other then getting the `ID`.
so you can properly just inject the `recipeId` right away to this VC.